lib/ret/storage.ex

```diff
@@ -61,19 +72,62 @@ defmodule Ret.Storage do
   end
 
   def fetch(id, key) when is_binary(id) and is_binary(key) do
-    fetch_blob(id, key, @expiring_file_path)
+    fetch_blob(id, key, expiring_file_path())
   end
 
   def fetch(%OwnedFile{owned_file_uuid: id, key: key}) do
-    fetch_blob(id, key, @owned_file_path)
+    fetch_blob(id, key, owned_file_path())
+  end
+
+  def fetch(%CachedFile{} = cached_file) do
+    fetch_at(cached_file, Timex.now() |> Timex.to_naive_datetime() |> NaiveDateTime.truncate(:second))
+  end
+
+  defp maybe_bump_accessed_at(%CachedFile{accessed_at: accessed_at} = cached_file, time) do
+    one_day_ago = Timex.shift(time, days: -1)
```

Comment:
Presumably this time constant is related specifically to the two-day expiration we've defined in cached_file.ex? In any case, it might be good to add a comment here explaining why this happens to use one day.